script.js

```diff
@@ -196,4 +300,6 @@ function restart() {
 
   Array.from(questions).forEach(item => item.classList.remove('question-active'));
   questions[0].classList.add('question-active');
+  preloader.classList.remove('none');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/voronkova-ab/-game-Who-want-to-be-a-millionaire-/pullRequest?prid=7740148)
```suggestion
  preloader.classList.remove("none");
```